addons/project/views/project_views.xml

```diff
@@ -1691,9 +1703,47 @@
             <field name="view_id" ref="view_task_all_calendar"/>
         </record>
 
+        <record id="action_view_all_task2" model="ir.actions.act_window">
```

Comment:
I'm not very happy that we have almost the same names for the "My Tasks" action and views, but I'm not sure what can be done about it. :confused:
I will ask if renaming the current ones is a possibility

Reply:
Yes we can, with an associated upgrade script